src/net/client/redundant.rs

```diff
@@ -54,24 +54,51 @@ const SMOOTH_N: f64 = 8.;
 /// Chance to probe a worse connection.
 const PROBE_P: f64 = 0.05;
 
-/// Avoid sending two requests at the same time.
-///
-/// When a worse connection is probed, give it a slight head start.
-const PROBE_RT: Duration = Duration::from_millis(1);
-
 //------------ Config ---------------------------------------------------------
 
 /// User configuration variables.
 #[derive(Clone, Copy, Debug, Default)]
 pub struct Config {
     /// Defer transport errors.
-    pub defer_transport_error: bool,
+    defer_transport_error: bool,
 
     /// Defer replies that report Refused.
-    pub defer_refused: bool,
+    defer_refused: bool,
 
     /// Defer replies that report ServFail.
-    pub defer_servfail: bool,
+    defer_servfail: bool,
+}
```

Comment:
Minor observation: as this code is marked unstable there changes cannot be breaking.

Reply:
The reason is consistency. For example in stream.rs we have:
pub fn set_response_timeout(&mut self, timeout: Duration) {
        self.response_timeout = RESPONSE_TIMEOUT.limit(timeout);
        self.streaming_response_timeout = self.response_timeout;
    }

It would be very confusing if the user has to figure out for each config field if the field is public and can be accessed directly, or private and one of the get/set function has to be used.